halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/Telemetry.java

```diff
@@ -31,7 +31,7 @@ public String getNodeName() {
     return "telemetry";
   }
 
-  private Boolean enabled = false;
+  private boolean enabled = false;
```

Comment:
I don't if there would be side effects for changing this to not allow nulls, what happens if an old hal config file which doesn't have `telemetry` section suddenly is loaded by this code?

Reply:
I think it'll be parsed as false and should behave correctly.